cli/features/support/env.ls

```diff
@@ -1,22 +1,26 @@
 require! {
+  'cucumber': {defineSupportCode}
+  './world': World
   'child_process'
   '../../../exosphere-shared' : {kill-child-processes, DockerHelper}
   'fs-extra' : fs
 }
 
 
-module.exports = ->
+defineSupportCode ({After, set-default-timeout, set-world-constructor}) ->
 
-  @set-default-timeout 2000
+  set-default-timeout 2000
+  set-world-constructor World
 
 
-  @After tags: ['~@e2e'], (scenario, done) ->
+  After tags: '~@e2e', (scenario, done) ->
```

Comment:
The tag syntax changed in cucumber 2.0. `~@e2e` should become `not @e2e`. Same for `~@docker-cleanup` below